ee/app/livechat-enterprise/server/lib/jobs.ts

```diff
@@ -0,0 +1,10 @@
+import { transferToNewAgent } from './Helper';
+
+export const autoTransferVisitorJob = async ({ attrs: { data } }: any = {}): Promise<void> => {
```

Comment:
This method is called inside the `Monitor` class only, why do you need another file to place and export it rather than implementing it as a private method in the class?